nginx.conf.src

```diff
@@ -60,22 +70,19 @@ http {
     location =/developer {
       return 302 https://open.gsa.gov/api/dap/;
     }
-    
+
     location / {
-      root <%= ENV["APP_ROOT"] %>/public;
```

Comment:
APP_ROOT may have been used by other orgs using this repo.

Hardcoding it should work for cloud.gov.

Reply:
I did a search in the repo and didn't see it referenced anywhere else in the code outside of the nginx.conf. So I think we're good here.